webservices/tasks/__init__.py

```diff
@@ -4,7 +4,7 @@
 
 from webservices.env import env
 from webservices.tasks import utils
-
+import ssl
```

Comment:
Does ssl pkg need to be in requirements.txt file?

Reply:
@pkfec thanks for your review! `ssl` is part of the [Python standard library](https://docs.python.org/3.7/library/ssl.html) so we don't need to/can't put it in the `requirements.txt` file.

Reply:
It's good to know what we don't need to put in the requirements.txt file. But if we can i think the SSL pkg is available here: https://pypi.org/project/ssl/

Reply:
Looks like that package was for python versions prior to 2.7 when `ssl` was added to the standard library.